src/main/java/com/it4045/pokemonviewer/enterprise/PokemonViewerController.java

```diff
@@ -13,9 +13,7 @@
 import org.springframework.web.bind.annotation.*;
 
 import java.io.IOException;
-import java.util.ArrayList;
```

Comment:
removed unused imports. I know how easy it is for these to pile up. I've learned over time to clean them up periodically. no big deal.